src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2751,7 +2757,7 @@ protected function generateTree($path, $intMargin, $mount=false, $blnProtected=t
 
 				if ($this->strPickerField)
 				{
-					$_buttons .= $this->getPickerInputField($currentEncoded, ($GLOBALS['TL_DCA'][$this->strPickerTable]['fields'][$this->strPickerField]['eval']['files'] || $GLOBALS['TL_DCA'][$this->strPickerTable]['fields'][$this->strPickerField]['eval']['filesOnly']) ? '' : ' disabled');
+					$_buttons .= $this->getPickerInputField($currentEncoded);
```

Comment:
Because that line is only executed if there are files, and if there are files they are always selectable. `blnFilesOnly` only defines that folders are not to be selected.